optimum/exporters/openvino/utils.py

```diff
@@ -158,3 +159,10 @@ def clear_class_registry():
     torch._C._jit_clear_class_registry()
     torch.jit._recursive.concrete_type_store = torch.jit._recursive.ConcreteTypeStore()
     torch.jit._state._clear_class_state()
+
+
+def save_config(save_dir, config):
```

Comment:
How about always calling your `save_config` instead of `model.config.save_pretrained`?

Reply:
done